lib/ssl/doc/src/ssl.xml

```diff
@@ -988,6 +988,15 @@ fun(srp, Username :: binary(), UserState :: term()) ->
       </desc>
     </datatype>
 
+    <datatype>
+      <name name="use_ktls"/>
+      <desc><p>Specifies whether to use linux kernel TLS offload</p>
+      <p>This is an experimental feature that could introduce a large performance boost.</p>
+      <p>Currently this feature only work for TLS 1.3 with AES_256_GCM_SHA384 cipher on certain linux kernel.</p>
+      <p>It also cannot be enabled when key_update_at is set.</p>
+      </desc>
+    </datatype>
+
```

Comment:
For now we would not want this be a ssl (TLS) option but a distribution option, that can only be used for the distribution.